crates/proto/proto/responses.proto

```diff
@@ -32,6 +32,12 @@ message NullifierUpdate {
     fixed32 block_num = 2;
 }
 
+message TransactionUpdate {
+    digest.Digest transaction_id = 1;
+    fixed32 block_num = 2;
+    fixed64 account_id = 3;
```

Comment:
Is there any reason why this is `fixed64` instead of `account.AccountId`?

Reply:
nope, changed it for account.AccountId